packages/@markuplint/file-resolver/src/ml-file/ml-file.ts

```diff
@@ -73,6 +74,16 @@ export class MLFile {
 		return '';
 	}
 
+	ignored(globPath: string | readonly string[]) {
+		if (typeof globPath === 'string') {
```

Comment:
Should `minimatch` be used if `globPath` is not an array?
Intuitively, I prefer the following code

```ts
const normalizedPaths = (typeof globPath === 'string' ? [globPath] : globPath).map(p => pathNormalize(p, true));
```